docs/core/whats-new/dotnet-9/runtime.md

```diff
@@ -93,6 +106,32 @@ static int Sum(int[] arr)
 
 The index variable, `i`, is 4 bytes in size. At the assembly level, 64-bit registers are typically used to hold array indices on x64, and in previous .NET versions, the compiler generated code that zero-extended `i` to 8 bytes for the array access, but continued to treat `i` as a 4-byte integer elsewhere. However, extending `i` to 8 bytes requires an additional instruction on x64. With IV widening, the 64-bit JIT compiler now widens `i` to 8 bytes throughout the loop, omitting the zero extension. Looping over arrays is very common, and the benefits of this instruction removal quickly add up.
 
+#### Loop counter variable direction
```

Comment:
This is fine here. As a general policy, I wonder if we should reserve "what's new" for functional changes and leave performance improvements for release notes and blog posts.

Reply:
@jkotas

Reply:
I would expect that the document titled "What's new in .NET 9 runtime" is going to cover all types of changes: functional, performance, security, diagnosability, reliability, major platform matrix updates, ... .

Reply:
Sounds good. I wanted a second opinion on that. That's a fine policy.